xarray/tests/test_dataarray.py

```diff
@@ -3681,6 +3681,20 @@ def test_to_dataframe_0length(self):
         assert len(actual) == 0
         assert_array_equal(actual.index.names, list("ABC"))
 
+    def test_to_dataframe__attrs(self):
+        arr = DataArray(
+            np.zeros((5, 5)),
+            name="test",
+            dims=("y", "x"),
+            coords={"y": np.arange(1, 6), "x": np.arange(2, 7)},
+            attrs={"long_name": "Description of data array", "_FillValue": -1},
+        )
+        df = arr.to_dataframe()
+        assert df[df.columns[0]].attrs == {
```

Comment:
Are these set per column or for the whole dataframe?

Reply:
Both actually. It is on the dataframe and the series.

Reply:
Should we test for both?

Reply:
I have a test for both in `xarray/tests/test_dataset.py`. I don't believe it is applicable in `xarray/tests/test_dataarray.py `.

Reply:
Forgive me if I'm confusing things.

How does pandas handle attrs? Here, it looks like it's the same on both the series and the dataframe — is that always the case? Or do we need to test both?

```

In [8]: df.attrs = dict(a=2)

In [9]: df
Out[9]:
     test
y x
1 2   0.0
  3   0.0
  4   0.0
  5   0.0
  6   0.0
2 2   0.0
  3   0.0
  4   0.0
  5   0.0
  6   0.0
3 2   0.0
  3   0.0
  4   0.0
  5   0.0
  6   0.0
4 2   0.0
  3   0.0
  4   0.0
  5   0.0
  6   0.0
5 2   0.0
  3   0.0
  4   0.0
  5   0.0
  6   0.0

In [10]: df.attrs
Out[10]: {'a': 2}

In [11]: df['test'].attrs
Out[11]: {'a': 2}
```

Reply:
Added more tests to ensure the expected behavior occurs.